modules/apps/calendar/calendar-web-test/src/testFunctional/macros/CalendarPermissions.macro

```diff
@@ -0,0 +1,68 @@
+definition {
+
+	var siteName = "Liferay";
```

Comment:
Is this variable being used? If it is, please change it's value to `var siteName = TestCase.getSiteName(siteName = "${siteName}");`. If it is not, you can delete.

Reply:
Done!